lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1164,21 +1164,25 @@ sub check_nutrition_data ($product_ref) {
 		}
 		# raise error if
 		# all values are identical
+		# and values (check first value only) are above 1 (see issue #9572)
 		#  OR
 		# all values but one - because sodium and salt can be automatically calculated one depending on the value of the other - are identical
-		# and values (check first value only) are above 1 (see issue #9572)
+		# and values (check salt (should not check sodium which could be lower)) are above 1 (see issue #9572)
 		# and at least 4 values are input by contributors (see issue #9572)
 		if (
 			(
-				($nutriments_values_occurences_max_value == scalar @major_nutriments_values)
+				(
+					$nutriments_values_occurences_max_value == scalar @major_nutriments_values
+					and (@major_nutriments_values[0] > 1)
```

Comment:
```suggestion
					and ($major_nutriments_values[0] > 1)
```

This currently causes a warning in the main branch:

Scalar value @major_nutriments_values[0] better written as $major_nutriments_values[0] at /opt/product-opener/lib/ProductOpener/DataQualityFood.pm line 1171.

Reply:
done.